man/src/oxide-config.1.md

```diff
@@ -0,0 +1,265 @@
+
+**M**
+: Meta key
+
+**A**
+: ALT key
+
+**C**
+: CONTROL key
+
+**S**
+: SHIFT key
+
+# COMMANDS
+
+Commands consist of a command and optional arguments.
+
+## COMMAND
+
+Move **args** [MOVEMENT]
+: Move window
+
+Focus **args** [MOVEMENT]
+: Move Focus
+
+Quit
+: Quit the window manager
+
+Kill
+: Kill the currently focused window
+
+Restart
+: Reloads the config and restarts components
+
+Layout **args** [LAYOUT]
+: Change the current layout
+
+GoToWorkspace **args** [WORKSPACE_ARGS]
+: Change the current workspace
+
+MoveToWorkspace **args** [WORKSPACE_ARGS]
+: Move the used window to a different workspace
+
+MoveToWorkspaceAndFollow **args** [WORKSPACE_ARGS]
+: Move the focused window to and select a different workspace
+
+Exec **args** <COMMAND>
+: Execute a given command
+
+Fullscreen
+: Toggle fullscreen mode for the focused window
+
+## ARGS
+
+Command arguments are necessary for the movement, the layout or to controll workspaces
+
+## MOVEMENT
+
+Left
+: Moves to the left
+
+right
+: Moves to the right
+
+## LAYOUT
+
+VerticalStriped
+: windows vertically next to each other
+
+HorizontalStriped
+: windows horizontally underneath each other
+
+None
+: if no argument is provided, the next layout is chosen
+
+## WORKSPACE_ARGS
+
+Next
+: next initialized workspace with a higher index than the current workspace
+
+Previous
+: next initialized workspace with a lower index than the current workspace
+
+Next_free
+: next available workspace with a higher index than the current workspace which is not initialized
+
+Index
+: workspace with the given index
+
+## ITERATIONS
+
+iter
+: iterates over given number in order to change
+
+# DEFAULT KEYBINDINGS
+
+Here is a short overview of the default keybindings
+
+Meta+Shift+e
+: quits the window manager
+
+Meta+Shift+r
+: restarts the window manager
+
+Meta+Shift+q
+: kills the current window
+
+h/l
+: direction keys (left/right)
+
+Meta+<direction>
+: changes the focus to the <direction> window
+
+Meta+Shift+<direction>
+: moves the window to the <direction>
+
+Meta+f
+: changes the current window to fullscreen
+
+Meta+u
+: switches to the next layout
+
+Meta+i
+: changes the layout to vertical
+
+Meta+Shift+i
+: changes to layout to horizontal
+
+Right/Left;
+: workspace navigation keys (next/previous)
+
+Meta+<workspace direction>
+: changes to the <workspace direction> workspace
+
+Meta+n
+: opens a new workspace
+
+Control+Meta+<workspace direction>
+: moves a window to the <workspace direction> workspace
+
+Control+Meta+n
+: opens a new workspace and moves the window to it
+
+Meta+Shift+<workspace direction>
+: moves the window to the <workspace direction> workspace and follows it
+
+Meta+Shift+n
+: creates a new workspace, moves the window to it and follows
+
+Control+Meta+Down
+: quits the workspace
+
+Meta+t
+: opens a new firefox window
+
+1/2/3/4/5/6/7/8/9;
+: workspace numbers
+
+Meta+<workspace number>
+: switches to workspace <workspace number>
+
+Control+Meta+<workspace number>
```

Comment:
the section in the `<thing>` is not rendered correctly. PLEASE compile this before pushing and look how the result is rendered :(

Reply:
Resolved this on branch doc/issue139-man-pages-config-statusbar-config . Please review the related pr.